custom_button/main.py

```diff
@@ -0,0 +1,46 @@
+import tkinter as tk
+from tkinter import *
+from CustomButton import CustomButton
+
+def changing_information(button_list):
+    for x in button_list:
+        x.get_button().config(
+            background=CustomButton.color,
+            activebackground=CustomButton.activebackground,
+        )
+        x.get_frame().place(
+            x=x.calculate_x_coordinate(),
+            width=CustomButton.width,
+            height=CustomButton.height,
+        )
+
+
+class App:
```

Comment:
Great idea! I really like it!
But what do you think about move this class into separate directory and prepare also documentation?